etc/org.opencastproject.speechtotext.impl.engine.WhisperCppEngine.cfg

```diff
@@ -54,3 +54,30 @@
 # This is done by Opencast itself before handing over the audio track.
 # Default: true
 #whispercpp.auto-encode=true
+
+
+## Optional settings for Voice Activity Detection (VAD)
+
+# Enable Voice Activity Detection (VAD)
+#whispercpp.vad=false
```

Comment:
Should we enable this by default? Currently, there is only the Silero model, so users don't really have options to choose from, like with the base Whisper model. Moreover, the Silero model is < 1mb, and we could easily always include it as a hard dependency. Opinions from RPMs/DEBs perspective @lkiesow, @gregorydlogan?

Reply:
Technically this would mean that you couldn't run with the old configuration anymore, which we shouldn't do on a release branch. But we break that rule at other places as well, so if the RM (@dennis531) is fine with that, I'm fine with that :shrug: 

In that case, we definitely shouldn't target `r/17.x`, though.

> Opinions from RPMs/DEBs perspective

I'll definitely package the model for OC18.

Reply:
I've packed with silero model as it's own, optional dependency. I'm fine making it a hard requires until there are more vad models, but I figured if you're not using VAD then you don't technically need the model. Don't much care though 🤷